### ISSUE TITLE
Verified Servers

### ISSUE THREAD

**Issue by ThatOneSponk** [Minehut/Meta#328]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
In Game
**Describe the feature**
When browsing the large array of Minehut servers, you'll find that a lot of the servers share distinct similarities. To prevent this, we should do our best to push servers that are well-made and unique. Featured servers was a great start, but I believe that even more can be done. The idea I have in mind would be "Verified Servers". It would follow a similar system to featured servers, but the server would stay verified for the server's lifespan. 
To apply for "Server Verification Status", you would fill out a form expressing why your server should be featured. Once that form is filled out, the server will be reviewed. Depending on many factors (Replayability, uniqueness, etc.,) the server would be considered for verification.

**Verified Server Benefits**
- Active 24/7 *(Does not shut down automatically)*
- 35 Max Player Slots
- Displayed in a special GUI. (Example: [Click here](https://imgur.com/2jjZKBG))

**How the feature is useful**
It will encourage the people who create servers on Minehut to produce servers that are unique and well made. This will also give users a list of servers with quality assurance to play on as opposed to finding something on the vast server list. This would keep users that are new to Minehut on for a longer duration of time since they aren't constantly replaying similar servers.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by ThatOneSponk** [comment]
> I believe that the idea of having featured servers on Minehut rather than verified servers is better.
> 
> A server shouldn't be verified or featured for the server's lifespan. I assume that you mean that once the server dies out it will be removed from the verified server list. At this point, why even verify the server in the first place? Monthly featured servers are sufficient for this as they have a defined time span for which they have perks for. I imagine that being featured for one month encourages the server owner and/or developers to add new features and make the server even more interesting so it could be featured in another couple of months or so.
> 
> Featured servers, as expressed above, give the quality assurance and uniqueness that players would want. When servers were featured, they were given a good amount of official endorsement when they were announced on the forums. Take [this](https://forums.minehut.com/topic/6653-gravestone/) post for example, the announcer went through the main points of the server and encouraged people to check it out. However, despite this quality assurance and uniqueness that was given by Minehut officially, a good amount of the featured servers never really made it past a week or two.
> 
> Something that could be improved from how it was done then was utilizing featured server slots on the server list (#56), this would give servers a lot more exposure than they were given when they were simply announced on the forums. This might also address the issue that I pointed out above about these featured servers not lasting a very long time. However, this was already suggested in this issue and I don't think it needs to be reinforced any further.
> 
> If verified servers were to ever become a thing, I believe they should be reserved specifically for official servers on Minehut that are maintained by Minehut staff members officially. Featured servers and verified servers in your suggestion pose no significant difference that would make one better than the other in your suggestion. To me, classifying community maintained servers that are recognized by Minehut as featured and classifying official servers that are actively maintained by Minehut staff as verified would make more sense.

I agree with a lot of your points, but the biggest benefit of having a larger amount of servers "featured" at once will give newer players a large variety of servers with quality assurance that they can play on. For example, if there is only a single featured server, the player may not be a fan of that genre and they'd be stuck sorting through servers that may not be as high of quality. I think removing the "until the server dies" point from my idea and replacing it with a specific timespan is reasonable as that is something that can't be easily determined along with potential repetitiveness.
 
But, to sum it up, having a larger amount of servers that are featured/verified will increase the likelihood of new players staying on the network as it will highly encourage well-made servers. I have seen multiple complaints from new users that a large majority of Minehut servers are the exact same which essentially discourages them from staying on the network for a long period of time.

*Essentially, the same featured server system but on a larger scale & with fewer perks*

**Comment by Sniped** [comment]
> But, to sum it up, having a larger amount of servers that are featured/verified will increase the likelihood of new players staying on the network as it will highly encourage well-made servers. I have seen multiple complaints from new users that a large majority of Minehut servers are the exact same which essentially discourages them from staying on the network for a long period of time.

I agree that there should be more than one featured server a month, maybe having 3-5 featured servers at a time per month. Though, a huge issue I saw with featured servers in general is finding the servers that differ from the rest of the Minehut community and the community's motivation to make and play these servers. I found that many of the servers that were featured declined despite their uniqueness, originality, and exposure from Minehut. Maybe there could be some further benefit to playing these servers?

**Comment by LukasmanMHdude** [comment]
^ preatty sure they were doing one a week or smth (or we could) to stagger the servers but still have 3 or 4 featured at one time.

**Comment by PundaChen** [comment]
Just an idea, three slots, one new server is added a week, each server stays on the list for 3 weeks

**Comment by LukasmanMHdude** [comment]
also would be nice if the featured slots worked more then just when minehut gets a sponser.

**Comment by MrEnxo** [comment]
@PundaChen Ideas is great and yea minehut just gets a sponsor and features them or just makes their own server and features them its a waste of space

**Comment by ThatOneSponk** [comment]
> Ideas is great and yea minehut just gets a sponsor and features them or just makes their own server and features them its a waste of space

Right, that's why featured servers should have their own separate slot in the GUI so they can preserve the 3 in the top right for Minehut events and whatnot.

> Maybe there could be some further benefit to playing these servers?

Giving benefits to those who play on the server or the creators themselves?
